reference/tax_rates_zones.v3.yml

```diff
@@ -60,8 +61,8 @@ paths:
                           locations:
                             - country_code: AR
                               subdivision_codes:
-                                - T
-                                - V
+                                - ARR
```

Comment:
I'm looking at Argentina subdivision codes https://www.iso.org/obp/ui/#iso:code:3166:AR . Would users specify "AR-R" or simply a letter like "R"?

Reply:
We should be using R. This is the modelling in bcapp_common db.

Reply:
Great, thanks, just changed it back to one letter